scripts/devnet_stress_test.py

```
@@ -0,0 +1,211 @@
+async def submit_query(
+    base_url: str, query_id: int, query: str
+) -> QueryResult:
+    start = time.perf_counter()
+    try:
+        payload = json.dumps({"query": query, "agent": "stress_test_agent"}).encode("utf-8")
+        req = urlrequest.Request(
+            f"{base_url}/api/inference",
+            data=payload,
+            headers={"Content-Type": "application/json"},
+            method="POST",
+        )
```

Ruff (0.15.11)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)